ark/utils/spatial_analysis_utils.py

```diff
@@ -84,34 +76,49 @@ def get_pos_cell_labels(analysis_type, pheno=None, current_fov_neighborhood_data
     Returns:
         list:
             List of all the positive labels"""
+    if(
+        thresh is None
+        or current_fov_channel_data is None
+        or cell_labels is None or current_marker is None
+    ):
+        raise ValueError("Incorrect arguments passed for analysis type")
+    # Subset only cells that are positive for the given marker
+    marker1posinds = current_fov_channel_data[current_marker] > thresh
+    # Get the cell labels of the positive cells
+    mark1poslabels = cell_labels[marker1posinds]
 
-    if not np.isin(analysis_type, ("cluster", "channel")).all():
-        raise ValueError("Incorrect analysis type")
+    return mark1poslabels
+
+
+def get_pos_cell_labels_cluster(pheno, current_fov_neighborhood_data):
+    """For cluster enrichment, finds positive labels that match the current phenotype
+    or identifies cells with positive expression values for the current marker
+    (greater than the marker threshold).
+
+    Args:
+        pheno (str):
+            the current cell phenotype
+        current_fov_neighborhood_data (pandas.DataFrame):
+            data for the current patient
+
+    Returns:
+        list:
+            List of all the positive labels"""
+
+    if pheno is None or current_fov_neighborhood_data is None:
+        raise ValueError("Incorrect arguments passed for analysis type")
+
+    # Subset only cells that are of the same phenotype
+    pheno1posinds = current_fov_neighborhood_data["FlowSOM_ID"] == pheno
+    # Get the cell labels of the cells of the phenotype
+    mark1poslabels = current_fov_neighborhood_data.iloc[:, 1][pheno1posinds]
 
-    if analysis_type == "cluster":
-        if pheno is None or current_fov_neighborhood_data is None:
-            raise ValueError("Incorrect arguments passed for analysis type")
-        # Subset only cells that are of the same phenotype
-        pheno1posinds = current_fov_neighborhood_data["FlowSOM_ID"] == pheno
-        # Get the cell labels of the cells of the phenotype
-        mark1poslabels = current_fov_neighborhood_data.iloc[:, 1][pheno1posinds]
-    else:
-        if(
-            thresh is None
-            or current_fov_channel_data is None
-            or cell_labels is None or current_marker is None
-        ):
-            raise ValueError("Incorrect arguments passed for analysis type")
-        # Subset only cells that are positive for the given marker
-        marker1posinds = current_fov_channel_data[current_marker] > thresh
-        # Get the cell labels of the positive cells
-        mark1poslabels = cell_labels[marker1posinds]
     return mark1poslabels
 
 
 def compute_close_cell_num(dist_mat, dist_lim, num, analysis_type,
```

Comment:
this was supposed to be fixed awhile ago, but got 'unfixed' from Jay and I's merge conflict.  I think `num` can just be computed from `cluster_ids` or `current_fov_channel_data`, and doesn't need to be passed in.